templates/step/edit.html.ep

```diff
@@ -140,15 +148,19 @@
                             </p>
                             <p>
                                 <!-- lists the specific properties -->
-                                <input type="checkbox" name="workaround" id="property_workaround"
-                                        %= 'checked' if $properties->{workaround}
-                                        >
+                                <input type="checkbox" name="workaround" id="property_workaround" >
                                 <label for="property_workaround">workaround</label>
                                 <%= help_popover('Help for the workaround property' =>
                                     'If a needle with the workaround property matches the match result is recorded as
                                     a \'soft-fail\'. It is suggested to mark the reason for this choice in either the
-                                    name of the needle (e.g. my-name-bsc1234-20161224) or with special tags')
+                                    name of the needle (e.g. my-name-bsc1234-20161224) or with special tags and
+                                    input the reason description in \'Description\' textbox.')
                                 %>
+                                <br>
```

Comment:
I don't care that much but the `<br>` which is not dynamically hidden/shown by your JavaScript code is still there. You could simply move it inside the subsequent span or use block-styled elements.

Reply:
I will move <br> inside the <span>

Reply:
I will move br inside the span